package.json

```diff
@@ -2,15 +2,15 @@
   "private": true,
   "scripts": {
     "build": "turbo run build",
-    "dev": "dotenv -- turbo run dev",
```

Comment:
Not sure what the call to `dotenv` was about. Maybe a mistake? AFAIK `dotenv` does not have a binary.